tests/data/packageVersion.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace ExamplePackageVersion;
+
+// TODO: phpunit/phpunit:<50 This has to be fixed when updating to phpunit 50.x
```

Comment:
actually I've done my check the other way around:
```
phpunit/phpunit:>=10
```
I think it reads better: "please phpstan, fail when phpunit meets this dependency"